specifyweb/frontend/js_src/lib/components/Attachments/Viewer.tsx

```diff
@@ -29,13 +29,15 @@ export function AttachmentViewer({
   related: [related, setRelated],
   showMeta = true,
   onViewRecord: handleViewRecord,
+  classNameProp,
```

Comment:
just call it `className`? the fact that it's a prop is clear as it can't be anything other than a prop so is redundant

The only case you would use prop in a name is if you do something like `inputProps` (i.e, in AutoComplete) to indicate that all those props would actually be passed to a different component